idc_collections/collex_metadata_utils.py

```diff
@@ -564,7 +564,8 @@ def get_metadata_solr(filters, fields, sources, counts_only, collapse_on, record
                 'facets': solr_facets,
                 'fqs': query_set,
                 'query_string': None,
-                'limit': 0,
+                'limit': record_limit,
```

Comment:
If you do this to the facet counting, the facets will ONLY show counts for those 2000 records. Facet counting should never be limited or offset--only record retrieval. While it's true that counts_only will ignore and override this value, it's still a potentially invisible bug I'd rather avoid.